pkg/k8s/events.go

```diff
@@ -137,7 +140,7 @@ func (source *EventSource) watch(cancel <-chan interface{}) {
 	}
 }
 
-// NewEventSource listens to kuberentes events in namespace. Call GetNewEvents
+// NewEventSource listens to kubernetes events in namespace. Call GetNewEvents
```

Comment:
I thought I was the only one who does typos of Kubernetes! :D p.s.: it's upper case :p